lib/Dav/Faces/FaceRoot.php

```diff
@@ -26,6 +26,10 @@ class FaceRoot implements ICollection, IMoveTarget {
 	private IMetadataManager $metadataManager;
 	private ITagManager $tagManager;
 	private IPreview $previewManager;
+	/**
+	 * @var \OCA\Recognize\Dav\Faces\FacePhoto[]
+	 */
+	private array $children;
```

Comment:
```suggestion
	private array $children = [];
```

Otherwise this would throw a type error on the check (ref https://php.watch/versions/7.4/typed-properties)